deploy/supervisor/production-background/huey-events.conf

```diff
@@ -0,0 +1,9 @@
+[program:huey-events]
+command=/home/cloo/doaj/bin/python /home/cloo/doaj/bin/huey_consumer.py -v portality.tasks.consumer_events_queue.events_queue
```

Comment:
updated deployment - changed binary paths /home/cloo/doaj/**venv/**bin/python and /home/cloo/doaj/**venv/**bin/huey_consumer.py